Assets/MRTK/Core/Utilities/DebugUtilities.cs

```diff
@@ -178,6 +178,7 @@ public static void DrawPoint(Vector3 point, Color color, float size = 0.05f)
         /// <summary>
         /// Draws a point with a rotation in the Scene window.
         /// </summary>
+        [System.Diagnostics.Conditional("UNITY_EDITOR")]
```

Comment:
What does this do? Is this method being called outside the editor and that's a problem? Should we clean up those calls instead of hiding that behavior?

Reply:
This is specifically a debug utility, but it causes the GUI system to tick every frame, even if the app doesn't use GUI. Debug.DrawLine only works on editor.

Reply:
Sounds good! I'm going to add a couple more too then, to save on method calls